Installation via pip on Raspbian Strech failled

Hi,

Here is my output during a `sudo pip install snowboy` on Raspbian Strech

```
sudo pip install snowboy
Collecting snowboy
  Using cached snowboy-1.2.0b1.tar.gz
Requirement already satisfied: PyAudio in /usr/local/lib/python2.7/dist-packages/PyAudio-0.2.11-py2.7-linux-armv7l.egg (from snowboy)
Building wheels for collected packages: snowboy
  Running setup.py bdist_wheel for snowboy ... error
  Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-4S_Hxf/snowboy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmp82jh2Opip-wheel- --python-tag cp27:
  running bdist_wheel
  running build
  Compiling snowboy...
  g++ -I../../ -O3 -fPIC -D_GLIBCXX_USE_CXX11_ABI=0 -std=c++0x  -shared snowboy-detect-swig.o \
  ../..//lib/rpi/libsnowboy-detect.a -L/usr/lib/python2.7/config-arm-linux-gnueabihf -L/usr/lib -lpython2.7 -lpthread -ldl  -lutil -lm  -Xlinker -export-dynamic -Wl,-O1 -Wl,-Bsymbolic-functions -lm -ldl -lf77blas -lcblas -llapack_atlas -latlas -o _snowboydetect.so
  snowboy-detect-swig.o: file not recognized: File format not recognized
  collect2: error: ld returned 1 exit status
  Makefile:65: recipe for target '_snowboydetect.so' failed
  make: *** [_snowboydetect.so] Error 1
  creating build
  creating build/lib.linux-armv7l-2.7
  creating build/lib.linux-armv7l-2.7/snowboy
  error: can't copy 'swig/Python/_snowboydetect.so': doesn't exist or not a regular file
  
  ----------------------------------------
  Failed building wheel for snowboy
  Running setup.py clean for snowboy
Failed to build snowboy
Installing collected packages: snowboy
  Running setup.py install for snowboy ... error
    Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-4S_Hxf/snowboy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-z5O3Qz-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    Compiling snowboy...
    g++ -I../../ -O3 -fPIC -D_GLIBCXX_USE_CXX11_ABI=0 -std=c++0x  -shared snowboy-detect-swig.o \
    ../..//lib/rpi/libsnowboy-detect.a -L/usr/lib/python2.7/config-arm-linux-gnueabihf -L/usr/lib -lpython2.7 -lpthread -ldl  -lutil -lm  -Xlinker -export-dynamic -Wl,-O1 -Wl,-Bsymbolic-functions -lm -ldl -lf77blas -lcblas -llapack_atlas -latlas -o _snowboydetect.so
    snowboy-detect-swig.o: file not recognized: File format not recognized
    collect2: error: ld returned 1 exit status
    Makefile:65: recipe for target '_snowboydetect.so' failed
    make: *** [_snowboydetect.so] Error 1
    creating build
    creating build/lib.linux-armv7l-2.7
    creating build/lib.linux-armv7l-2.7/snowboy
    error: can't copy 'swig/Python/_snowboydetect.so': doesn't exist or not a regular file
    
    ----------------------------------------
Command "/usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-4S_Hxf/snowboy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-z5O3Qz-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-4S_Hxf/snowboy/
```

Nope. Still having issues both through Pip and Pycharm.
```

$ pip install snowboy
Collecting snowboy
  Downloading snowboy-1.2.0b1.tar.gz (5.0MB)
    100% |████████████████████████████████| 5.0MB 318kB/s 
Requirement already satisfied: PyAudio in /usr/local/lib/python2.7/dist-packages (from snowboy)
Building wheels for collected packages: snowboy
  Running setup.py bdist_wheel for snowboy ... error
  Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-7QDEr9/snowboy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpwqOWpdpip-wheel- --python-tag cp27:
  running bdist_wheel
  running build
  Compiling snowboy...
  /bin/sh: 1: swig: not found
  expr: syntax error
  g++ -I../../ -O3 -fPIC -D_GLIBCXX_USE_CXX11_ABI=0 -std=c++0x  -shared snowboy-detect-swig.o \
  ../..//lib/ubuntu64/libsnowboy-detect.a -L/usr/lib/python2.7/config-x86_64-linux-gnu -L/usr/lib -lpython2.7 -lpthread -ldl  -lutil -lm  -Xlinker -export-dynamic -Wl,-O1 -Wl,-Bsymbolic-functions -lm -ldl -lf77blas -lcblas -llapack_atlas -latlas -o _snowboydetect.so
  snowboy-detect-swig.o: file not recognized: File format not recognized
  collect2: error: ld returned 1 exit status
  Makefile:65: recipe for target '_snowboydetect.so' failed
  make: *** [_snowboydetect.so] Error 1
  creating build
  creating build/lib.linux-x86_64-2.7
  creating build/lib.linux-x86_64-2.7/snowboy
  error: can't copy 'swig/Python/_snowboydetect.so': doesn't exist or not a regular file
  
  ----------------------------------------
  Failed building wheel for snowboy
  Running setup.py clean for snowboy
Failed to build snowboy
Installing collected packages: snowboy
  Running setup.py install for snowboy ... error
    Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-7QDEr9/snowboy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-OaVVxB-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    Compiling snowboy...
    /bin/sh: 1: swig: not found
    expr: syntax error
    g++ -I../../ -O3 -fPIC -D_GLIBCXX_USE_CXX11_ABI=0 -std=c++0x  -shared snowboy-detect-swig.o \
    ../..//lib/ubuntu64/libsnowboy-detect.a -L/usr/lib/python2.7/config-x86_64-linux-gnu -L/usr/lib -lpython2.7 -lpthread -ldl  -lutil -lm  -Xlinker -export-dynamic -Wl,-O1 -Wl,-Bsymbolic-functions -lm -ldl -lf77blas -lcblas -llapack_atlas -latlas -o _snowboydetect.so
    snowboy-detect-swig.o: file not recognized: File format not recognized
    collect2: error: ld returned 1 exit status
    Makefile:65: recipe for target '_snowboydetect.so' failed
    make: *** [_snowboydetect.so] Error 1
    creating build
    creating build/lib.linux-x86_64-2.7
    creating build/lib.linux-x86_64-2.7/snowboy
    error: can't copy 'swig/Python/_snowboydetect.so': doesn't exist or not a regular file
    
    ----------------------------------------
Command "/usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-7QDEr9/snowboy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-OaVVxB-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-7QDEr9/snowboy/
```

--- a/.gitignore
+++ b/.gitignore
@@ -50,3 +50,4 @@ Snowboy.pm
 
 /dist
 **/snowboy.egg-info
+/.idea

--- a/examples/Python3/snowboydecoder.py
+++ b/examples/Python3/snowboydecoder.py
@@ -2,7 +2,7 @@
 
 import collections
 import pyaudio
-import snowboydetect
+from . import snowboydetect
 import time
 import wave
 import os

--- a/setup.py
+++ b/setup.py
@@ -1,18 +1,21 @@
 import os
+import sys
 from setuptools import setup, find_packages
 from distutils.command.build import build
 from distutils.dir_util import copy_tree
 from subprocess import call
 
 
+py_dir = 'Python' if sys.version_info[0] < 3 else 'Python3'
+
 class SnowboyBuild(build):
 
     def run(self):
 
         cmd = ['make']
-
+        swig_dir = os.path.join('swig', py_dir)
         def compile():
-            call(cmd, cwd='swig/Python')
+            call(cmd, cwd=swig_dir)
 
         self.execute(compile, [], 'Compiling snowboy...')
 
@@ -20,7 +23,7 @@ class SnowboyBuild(build):
         self.mkpath(self.build_lib)
         snowboy_build_lib = os.path.join(self.build_lib, 'snowboy')
         self.mkpath(snowboy_build_lib)
-        target_file = 'swig/Python/_snowboydetect.so'
+        target_file = os.path.join(swig_dir, '_snowboydetect.so')
         if not self.dry_run:
             self.copy_file(target_file,
                            snowboy_build_lib)
@@ -42,8 +45,8 @@ setup(
     maintainer_email='snowboy@kitt.ai',
     license='Apache-2.0',
     url='https://snowboy.kitt.ai',
-    packages=find_packages('examples/Python/'),
-    package_dir={'snowboy': 'examples/Python/'},
+    packages=find_packages(os.path.join('examples', py_dir)),
+    package_dir={'snowboy': os.path.join('examples', py_dir)},
     py_modules=['snowboy.snowboydecoder', 'snowboy.snowboydetect'],
     package_data={'snowboy': ['resources/*']},
     zip_safe=False,

